examples/dreamcast/pvr/pvrmark_strips_direct/pvrmark_strips_direct.c

```diff
@@ -69,17 +75,18 @@ void setup(void) {
     pvr_poly_compile(&hdr, &cxt);
 }
 
-int oldseed = 0xdeadbeef;
+
+#define nextnum() seed = seed * 1164525 + 1013904223;
+#define getnum(mn) (seed & ((mn) - 1))
+
```

Comment:
I know you didn't write this bit, but since they're always used in a pair, it'd make sense and declutter a lot to combine them. If you don't wanna though, that's fine too, no need to clean up everything just because you want to clean up part of it :)